client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -38,75 +68,129 @@ const SubmitButton = styled(FormButton)`
 `
 
 interface IProps {
+  ballot: IBallot
   contests: IContest[]
   interpretations: IBallotInterpretation[]
   setInterpretations: (interpretations: IBallotInterpretation[]) => void
-  previousBallot: () => void
   confirmSelections: (interpretations: IBallotInterpretation[]) => void
+  submitBallotNotFound: () => void
+  previousBallot: () => void
 }
 
 const BallotAudit: React.FC<IProps> = ({
+  ballot,
   contests,
   interpretations,
   setInterpretations,
   confirmSelections,
+  submitBallotNotFound,
```

Comment:
what do you think about renaming this `confirmBallotNotFound` to match `confirmSelections`?

Reply:
:100: